### ISSUE TITLE
Please stop whitelisting trackers

### ISSUE THREAD

**Issue by jawz101** [EFForg/privacybadgerfirefox#560]
If something doesn't honor DNT it should be blocked.  That's the whole point.  This addon can't turn into Swiss Cheese simply because trackers are a commenting system or whatever on a site.  People can whitelist stuff if they want.  Otherwise I have to install something like Self-Destructing Cookies or something just to clean up after the stuff people get added to the allowed list.

**Comment by anonsubmitter** [comment]
Sorry, I legitimately just thought that the issues I reported were site bugs and didn't think any further than that. When I reported issues like this last year nobody had any problems with it.
I usually report stuff on the EasyList Adblock filter forum and it's more of a functionality driven mindset over there. I should have figured out the priorities of this community on my own. I mean, it's even in the name: privacy.

TL;DR: I messed up and I'll try to learn from it. Sorry.

**Comment by jawz101** [comment]
I don't know.  I mean, you may identify a pattern in some instances.  Say, if 3 sites are indeed owned by the same company there could be a reason content is identified as 3rd party if it's shared amongst the sites.  I just have the  mindset this add-on relies on heuristic to determine what to blacklist.  I mean, Disqus is widely used and feels legit because it serves one purpose but then it's not a free service for website developers out of the good nature of an organization.  Any time you see it on a page it's tracking which sites you visited so they can track your interests... it's really up to you but I just question why something is being blocked when the app is monitoring how many sites stuff shows on.  This isn't an add-on like Adblock Plus which is built around a blocklist and exceptions.  I've had about 450 potential 3rd party tracking domains logged in the app and my browsing isn't affected.  It's pretty crazy.  Still, they're only potentials because they've been 3rd party at least once.  Until they appear on 3 separate website domains they're still allowed to do what they do.

It's like fonts.googleapis.com https://developers.google.com/fonts/docs/getting_started (or whatever it is) is a font Google puts out that developers can use to "make the web beautiful."  It's Google- put out a neat font but really it's a tracker.

**Comment by anonsubmitter** [comment]
I think I'll do what I did before and report site bugs that I see and let the developers of Privacy Badger decide if something should be whitelisted or not. Just because I report it doesn't mean they'll whitelist it. (I've had some of my site bug reports rejected in the past - rightfully so - because the sites I reported didn't have any privacy policy.)

**Comment by cooperq** [comment]
Please do report all site bugs! We see them all, though we may choose to not fix some of them we like to know about them either way!

**Comment by anonsubmitter** [comment]
Ok, I've reported all of them and I will report future site bugs when they appear.
